listenbrainz_spark/stats/user/tests/test_utils.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from listenbrainz_spark.stats.user import utils as u1
```

Comment:
utils as u1 is probably not a good idea. How about this:
`import listenbrainz_spark.stats.user`
Then use, *user.utils* in your code.